### DISCUSSION TITLE
Using ani cli in rofi ?

### DISCUSSION THREAD

**Discussion by ghost** [pystardust/ani-cli#1024] [General] [answered]
Is there a way to make the ani cli script work in rofi so that I can easily execute the script with my keybindings and use it without opening my terminal ?

**Comment by 71zenith** [comment, marked as answer]
use `env ANI_CLI_EXTERNAL_MENU=1 ani-cli` to launch

**Reply by ghost** [reply]
> use `env ANI_CLI_EXTERNAL_MENU=1 ani-cli` to launch

How can I launch rofi with an environment variable ?

**Reply by 71zenith** [reply]
just launch ani-cli from rofi
it will detect if its running from exteranal launcher

**Reply by ghost** [reply]
nvm got it
